### PR TITLE
Updating Master with Develop branch

### PR DESCRIPTION
**Pull request by gioneill**
Merging remaining commits not yet on Master branch.

### CODE REVIEW

File: simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java [hunk 1]
```diff
@@ -526,6 +526,17 @@ else if (id == 1) {
     }
   }
 
+  @Override protected void onPause()
+  {
+    super.onPause();
+
+    final SimplifiedReaderAppServicesType rs = Simplified.getReaderAppServices();
+
+    if (this.book_id != null && this.current_location != null) {
```
Comment (daryanypl) [inline simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java]: Is bookmarks how we keep the last-read place in the book?  Would you mind adding a doc string explaining it?

File: simplified-app-shared/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
 
   <!-- EULA strings -->
   <string name="eula_disagree">Disagree</string>
   <string name="eula_agree">Agree</string>
+  <string name="eula_checkbox">I agree to the terms of the End User License Agreement.</string>
+
+  <!-- Age Verification strings -->
+  <string name="age_verification_title">Age Verification</string>
+  <string name="age_verification_question">You must be 13 years of age or older to download a book from the collection.\nHow old are you?</string>
+  <string name="age_verification_changed">If you are under 13, all content downloaded to My Books will be removed.\nHow old are you?</string>
```
Comment (daryanypl) [inline simplified-app-shared/src/main/res/values/strings.xml]: all content, or all YA content?

File: simplified-app-shared/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
+  <string name="age_verification_13_older">13 or Older</string>
+  <string name="age_verification_13_younger">Under 13</string>
+  <string name="age_verification_13_checkbox">I am 13 years of age or older.</string>
+  <string name="age_verification_too_young">You are not old enough to sign up for a library card.r</string>
```
Comment (daryanypl) [inline simplified-app-shared/src/main/res/values/strings.xml]: is the "card.r" a typo?  and... gosh we're harsh.

File: simplified-app-shared/src/main/java/org/nypl/simplified/app/catalog/CatalogFeedActivity.java [hunk 4]
```diff
@@ -962,13 +962,14 @@ public void showAgeCheckAlert() {
       final AlertDialog.Builder alert = new AlertDialog.Builder(CatalogFeedActivity.this);
 
       // Setting Dialog Title
-      alert.setTitle("Age Verification");
+      alert.setTitle(R.string.age_verification_title);
 
       // Setting Dialog Message
-      alert.setMessage("You must be 13 years of age or older to download a book from the collection.\nHow old are you?");
+      //alert.setMessage("You must be 13 years of age or older to download a book from the collection.\nHow old are you?");
```
Comment (mattniehoff) [inline simplified-app-shared/src/main/java/org/nypl/simplified/app/catalog/CatalogFeedActivity.java]: We don't need to keep this commented out line.

File: simplified-app-shared/src/main/res/values/strings.xml (hunk 2 above)
Comment (mattniehoff) [reply simplified-app-shared/src/main/res/values/strings.xml]: If this comes back as true, it looks like it calls reloadCatalog with a true parameter for deleting the books in the catalog.

I don't know how that code works on the back end.

Comment (gioneill) [reply simplified-app-shared/src/main/res/values/strings.xml]: We serve two completely different OPDS feeds based on the age, so I guess "all content". Any time you decide to change your age in settings, the confirmation informs the user that they will lose their current content and start over with the new feed.

File: simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java (hunk 1 above)
Comment (gioneill) [reply simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java]: Yes I will add that, just not in this PR. I've made a note to myself.

File: simplified-app-shared/src/main/res/values/strings.xml (hunk 3 above)
Comment (gioneill) [reply simplified-app-shared/src/main/res/values/strings.xml]: Typo!

File: simplified-app-shared/src/main/java/org/nypl/simplified/app/MainSettingsAccountActivity.java [hunk 5]
```diff
@@ -579,17 +579,39 @@ public void onClick(final DialogInterface dialog, final int which) {
...
+      in_signup.setOnClickListener(
+        new OnClickListener() {
+          @Override
+          public void onClick(
+            final @Nullable View v) {
+            final Intent cardcreator = new Intent(MainSettingsAccountActivity.this, CardCreatorActivity.class);
+            startActivity(cardcreator);
+          }
+        });
+      in_signup.setText("Sign Up");
```
Comment (mattniehoff) [inline simplified-app-shared/src/main/java/org/nypl/simplified/app/MainSettingsAccountActivity.java]: line 591 should be deleted, 592 does the same thing and references the string resource file.

**Comment by gioneill** [comment]
Changes implemented in commit: d9de16e on branch 'gradle-conversion'

**Pull request closed without merging**